docs/content/manual/_incoming/2176-airgap-private-registry-rancher-agent-image.md

```diff
@@ -0,0 +1,185 @@
+1. Apply following yaml in file 
+        ```
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+        name: myregistry-nginx
+        spec:
+        selector:
+            matchLabels:
+            image: myregistry-nginx
+        template:
+            metadata:
+            labels:
+                image: myregistry-nginx
+            spec:
+            containers:
+                - imagePullPolicy: Always
+                image: myregistry.local:5000/nginx:latest
+                name: nginx
+        ```
+        ```
+        harvester-node-0:~ # kubectl apply -f deploy.yaml
+        deployment.apps/myregistry-nginx created
+        
+        ```
+1. search deployments -> default namespace -> myregsistry-nginx -> describe -> shift + g
+1. Check the nginx can be deployed. 
+
+### Test Plan 2
+1. Click the `default value` to update the Harvester containerd-registry setting to default empty 
+    ![image](https://user-images.githubusercontent.com/29251855/193788085-0b1648d7-ef0e-495e-8898-4f743028f94c.png)
+1. In K9s
+1. Check content in cattle-system/harvester-containerd-registry secret is changed.
+1. Check there are new jobs to auto apply the new containerd-registry setting.
+1. Search secrets -> containerd -> e 
+1. Clean registry.yaml data field
+    ```
+    # Please edit the object below. Lines beginning with a '#' will be ignored,
+    # and an empty file will abort the edit. If an error occurs while saving this file will be
+    # reopened with the relevant failures.
+    #
+    apiVersion: v1
+    data:
+        registries.yaml: ""
+    kind: Secret
+    metadata:
+        creationTimestamp: "2022-10-04T08:51:47Z"
+        name: harvester-containerd-registry
+        namespace: cattle-system
+        resourceVersion: "42088"
+        uid: 8445547f-430a-474e-9783-dc6b16914eb7
+    type: Opaque
+    ```
+1. Search -> deployments -> find nginx-1.2.2
+1. Check cannot pull nginx-1.2.2 image
+    ![image](https://user-images.githubusercontent.com/29251855/193790572-d5f5c92e-cbb9-4528-9c01-748d37940bc1.png)
+
+
+### Test Plan 3
+1. Add back the Harvester containerd-registry setting to use private registry again 
+    ![image](https://user-images.githubusercontent.com/29251855/193790937-a702730e-cb57-4f16-8950-76372a1859e3.png)
+1. Check content in cattle-system/harvester-containerd-registry secret is changed.
+    ![image](https://user-images.githubusercontent.com/29251855/193791120-755347db-46d7-427b-9e17-adb77025b7c8.png)
+1. Check there are new jobs to auto apply the new containerd-registry setting.
+    ![image](https://user-images.githubusercontent.com/29251855/193791283-40985e8a-f450-4766-9f0c-68e708c643cb.png)
+1. Check nginx 1.22 deployment is automatically back to running.
+    ![image](https://user-images.githubusercontent.com/29251855/193791385-5107385d-66c5-44a3-a298-82c44af929c7.png)
+
+## Expected Results
+### Test Result 1
```

Comment:
I think _expected results_ should be in the same section of Test, separate them would cause unnecessary scrolling when doing the test. (same problem in 2734, 2665-2892)

Reply:
Thanks for the check. I have remove the separate test result section of #2176 , #2734 and #2665-#2892
And combine the test result steps in #2734